pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -1150,7 +1150,7 @@ cdef class _Timedelta(timedelta):
         """
         Format Timedelta as ISO 8601 Duration like
         ``P[n]Y[n]M[n]DT[n]H[n]M[n]S``, where the ``[n]`` s are replaced by the
-        values. See https://en.wikipedia.org/wiki/ISO_8601#Durations
+        values. See https://en.wikipedia.org/wiki/ISO_8601#Durations.
```

Comment:
@datapythonista I'm never sure what to do with URLs at the end of a sentence.  Has this been discussed?

Reply:
@jbrockmendel I wasn't sure either, but this seem to have trigger the error for the `Timedelta.isoformat` docstring.

@datapythonista updated my branch and the CI still failed.

Reply:
@sparalic can you build the html page of that docstring? IIRC you can build that page only with `./doc/make.py --single=pandas.Timedelta.isoformat` (building all the docs takes some time). This way we can see if the dot is added to the url link and breaks it, or if Sphinx is smart enough and does what we want.

Looks like there are tests that check those docstrings, you'll have to update them to pass the CI. Let me know if you need help finding the errors in the CI logs...

Thanks!

Reply:
@datapythonista I was able to sucessfylly build the html page for `Timedelta.isoformat` without any warnings. So it seems like Sphinx was smart enough to ignore the period on the end.

Reply:
Can you open it with a browser, and paste here a screenshot? I don't think Sphinx will warn, it could just create a link with the period in it, that is broken.

Reply:
The link is created correctly.